app/views/shared/_new_evidence_checklist.html.haml

```diff
@@ -0,0 +1,10 @@
+- unless @claim.evidence_checklist_ids.empty?
+  .govuk-summary-card
+    .govuk-summary-card__title-wrapper
+      %h2.govuk-summary-card__title= (t('shared.evidence_checklist.caption'))
+    .govuk-summary-card__content
+      = govuk_summary_list do
+        %ul(class="govuk-!-margin-0")
+        - DocType.find_by_ids(@claim.evidence_checklist_ids).each do |dt|
+          %div
+            = dt.name
```

Comment:
`govuk_summary_list` is creating a description list (`<dl>`) so the elements inside should be a term (`<dt>`) or description (`<dd>`). I think that the spacing helper is unnecessary if you do this so it is simpler, like this:

```suggestion
      = govuk_summary_list do
        - DocType.find_by_ids(@claim.evidence_checklist_ids).each do |dt|
          %dt #{dt.name}
```

Sorry, I should have spotted this when I commented on the unnecessary `%dl` line.

Reply:
Oh I see- that makes sense! Thanks Joe.